docs/.vitepress/config.ts

```diff
@@ -241,7 +241,41 @@ export default ({ mode }: { mode: string }) => {
               },
               {
                 text: 'Browser Mode',
-                link: '/guide/browser',
+                link: '/guide/browser/',
+                collapsed: true,
+                items: [{
+                  text: 'Headless',
```

Comment:
I don't think this should be in a separate page

Reply:
done